benchmark_runner/common/oc/oc.py

```diff
@@ -82,26 +75,29 @@ def get_worker_disk_ids(self):
         """
         workers_disk_ids = []
         if self.__worker_disk_ids:
-            workers_disk_ids = [disk_id for disk_list in self.__worker_disk_ids.values() for disk_id in disk_list]
+            for node, disk_ids in self.__worker_disk_ids.items():
+                for disk_id in disk_ids:
+                    workers_disk_ids.append(disk_id)
         return workers_disk_ids
 
+    def get_pv_disk_ids(self):
+        """
+        This method returns list of pv disk ids
+        """
+        pv_ids = self.run(f"{self.__cli} get pv -o jsonpath={{.items[*].metadata.annotations.'storage\.openshift\.com/device-id'}}")
+        return [pv[len(self.__worker_disk_prefix):] for pv in pv_ids.split()]
```

Comment:
Actually strip off the prefix (not just by length, by content).

Reply:
Again, don't remove the length of the prefix, actually strip the prefix.

Reply:
@RobertKrawitz updated